src/aatoolbox/config/countryconfig.py

```diff
@@ -81,12 +81,54 @@ def _set_region_code(cls, values) -> dict:
         return values
 
 
+class UsgsNdviConfig(BaseModel):
+    """USGS NDVI configuration.
+
+    Parameters
+    ----------
+    area_name : str
+        Name of the USGS NDVI coverage area the country belongs to.
+        Needed to download the regional NDVI data.
+    """
+
+    area_name_mapping: Dict[str, Tuple[str, str]] = {
+        "north-africa": ("africa/north", "na"),
+        "east-africa": ("africa/east", "ea"),
+        "southern-africa": ("africa/southern", "sa"),
+        "west-africa": ("africa/west", "wa"),
+        "central-asia": ("asia/centralasia", "cta"),
+        "yemen": ("asia/middleeast/yemen", "yem"),
+        "central-america": ("lac/camcar/centralamerica", "ca"),
+        "hispaniola": ("lac/camcar/caribbean/hispaniola", "hi"),
+    }
+    area_name: str
+
+    @validator("area_name")
+    def area_name_valid(cls, v, values) -> str:
+        """Check that area_name is valid."""
+        valid_area_names = values["area_name_mapping"].keys()
+        if v not in valid_area_names:
+            raise ValueError(
+                f"Invalid area name: {v}. "
+                f"Should be one of {', '.join(valid_area_names)}"
+            )
+        return v
+
+    @root_validator(pre=False, skip_on_failure=True)
+    def _set_area_codes(cls, values) -> dict:
+        """Set NDVI url and prefix from area."""
+        area_values = values["area_name_mapping"]["area_name"]
+        values["area_url"], values["area_prefix"] = area_values
```

Comment:
very small, but would combine this to one line, i.e. uneccessary to create `area_values`